contribs/gmf/src/lidarprofile/Utils.js

```diff
@@ -260,13 +260,7 @@ export default class {
     // Draw the profile canvas (the points) into the new canvas.
     const profileCanvas = d3select('#gmf-lidarprofile-container .lidar-canvas');
     const profileCanvasEl = /** @type {HTMLCanvasElement} */ (profileCanvas.node());
-    ctx.drawImage(
-      profileCanvasEl,
-      margin.left,
-      margin.top,
-      w - (margin.left + margin.right),
-      h - (margin.top + margin.bottom)
-    );
+    ctx.drawImage(profileCanvasEl, margin.left, margin.top);
```

Comment:
Is it the change that actually fixes the margin issue?

Reply:
Yes